src/services/__tests__/addProducts.test.ts

```diff
@@ -0,0 +1,32 @@
+import { addProducts } from '../index';
+import db from '../../models';
+import { Product } from '../../types';
+
+jest.mock('../../models', () => ({
```

Comment:
Please use this code as a reference for the https://github.com/shamanms/vash-lavash-api/pull/4/files#r968830395